releasenotes/notes/opt-result-update-cdb157f6a67483bb.yaml

```diff
@@ -1,3 +1,20 @@
+features:
+  - |
+    ``MinimumEigenOptimizationResult`` now exposes properties: ``samples`` and
+    ``eigensolver_result``. The latter is obtained from the underlying algorithm used by the
+    optimizer and specific to the algorithm.
+    ``RecursiveMinimumEigenOptimizer`` now returns an instance of the result class
+    ``RecursiveMinimumEigenOptimizationResult`` which in turns may contains intermediate results
+    obtained from the underlying algorithms. The dedicated result class exposes properties
+    ``replacements`` and ``history`` that are specific to this optimizer. The depth of the history
+    is managed by the ``history`` parameter of the optimizer.
+  - |
+    ``GroverOptimizer`` returns now an instance of ``GroverOptimizationResult`` and this result
```

Comment:
```suggestion
    ``GroverOptimizer`` now returns an instance of ``GroverOptimizationResult`` and this result
```
Since the one above have now returns this keeps things consistent